codemp/qcommon/files.cpp

```diff
@@ -263,8 +269,28 @@ typedef struct qfile_us {
 } qfile_ut;
 
 typedef struct fileHandleData_s {
+	fileHandleData_s() :
+			handleFiles({}),
+			handleSync(qfalse),
+			handleAsync(qfalse),
+			writerThread(nullptr),
+			closed(qfalse),
+			fileSize(0),
+			zipFilePos(0),
+			zipFileLen(0),
+			zipFile(qfalse) {
+		ospath[0] = '\0';
+		name[0] = '\0';
+	}
```

Comment:
[Just use in-class initializers](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c48-prefer-in-class-initializers-to-member-initializers-in-constructors-for-constant-initializers)?